content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -1,62 +1,69 @@
 ---
 linktitle: Creating Projects Using the CLI
 title: Creating Projects Using the Command Line Interface
-description: Learn how to create and build new Open 3D Engine (O3DE) projects from the default project template using the CLI.
+description: Learn how to use the command line interface (CLI) to create and build new Open 3D Engine (O3DE) projects from the default project template.
 weight: 200
 toc: true
 ---
 
-The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using the command line interface (CLI).
+The tutorial on this page and the video below provide an introduction to project configuration and building in **Open 3D Engine (O3DE)** using the `o3de` command line interface (CLI) tool. The **O3DE CLI** is a Python script used for project configuration. Invoke it from the `scripts` directory in your O3DE install. For a complete guide to O3DE CLI commands, refer to the [Project Configuration CLI Reference](/docs/user-guide/project-config/cli-reference).
 
 The instructions here and in the video guide you through the following steps:
 
-* Create a new O3DE project using the CLI.
+* Create a new O3DE project using the O3DE CLI.
 * Create Visual Studio project files in your O3DE project directory.
 * Build your O3DE project.
 
-At the end of the tutorial you'll have a new O3DE project based on the default project template, opened in the **O3DE Editor**.
+At the end of the tutorial, you'll have a new O3DE project based on the **standard** template (the default project template).
```

Comment:
I suggest rewording this to improve the flow. I liked the way you worded it in `docs/welcome-guide/create/creating-projects-using-project-manager/` line 15. 

Also, I suggest removing the bold-style around "standard". If "standard" is the name of the project template, then use quotation marks around it. 

```suggestion
At the end of the tutorial, you'll have a new O3DE project based on the default "standard" project template.
```

Reply:
Paul and I have gone back and forth on this, but in the end I think I slightly prefer yours and my word choice. Changing it for the last time! :)